.github/dependabot.yml

```diff
@@ -0,0 +1,13 @@
+# https://docs.github.com/en/code-security/dependabot/working-with-dependabot/keeping-your-actions-up-to-date-with-dependabot#example-dependabotyml-file-for-github-actions
+# https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#about-the-dependabotyml-file
+
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+    reviewers:
+      - "mbta/ctd-infrastructure"
```

Comment:
This makes sense to me with one caveat--I'm not sure @mbta/ctd-infrastructure is the right group to review dependabot PRs in the actions repo. Any thoughts, Ian or Paul?

Reply:
I would agree with that. Maybe we should have an `@mbta/tech-leadership` group to round-robin between the leads?

Reply:
I just created @mbta/tid-tech-leads for this purpose. @thecristen can you update the PR?

Reply:
Done